src/test/java/org/ow2/petals/flowable/ServiceProviderVacationProcessTest.java

```diff
@@ -115,14 +114,12 @@ public static void assertTechnicalMonitoringMetrics()
         // process instances
         final Long[] vacationInstances = metrics.get("vacationRequest");
         assertEquals(0, vacationInstances[0].longValue());
-        
+
         /*
-         * 5 process instances in state 'Pending', the ones created by unit tests:
-         * - userTaskRequest_EmptyProcessInstanceIdValue,
-         * - userTaskRequest_EmptyUserIdValue,
-         * - userTaskRequest_NoUserIdValue,
-         * - userTaskRequest_NoProcessInstanceIdValue,
-         * - userTaskRequest_TaskCompletedFault
+         * 5 process instances in state 'Pending', the ones created by unit tests: -
+         * userTaskRequest_EmptyProcessInstanceIdValue, - userTaskRequest_EmptyUserIdValue, -
+         * userTaskRequest_NoUserIdValue, - userTaskRequest_NoProcessInstanceIdValue, -
+         * userTaskRequest_TaskCompletedFault
```

Comment:
Formatting is broken, you should keep new lines here.

Reply:
Was fixed merging changes